requirements-dev.txt

```diff
@@ -110,7 +110,7 @@ docker==7.1.0
     # via testcontainers
 email-validator==2.2.0
     # via fastapi
-fastapi[all]==0.112.2
+fastapi[all]==0.115.5
```

Comment:
Wondering the rationale for the package updates here?

Reply:
@dancoates FYI

Reply:
As you can see there is a new security issue (failed check) since Friday, when it was OK: 'Found 1 known vulnerability in 1 package'
Personally I would be reluctant to merge with those red crosses :-)